flexget/utils/imdb.py

```diff
@@ -37,6 +37,26 @@ def is_imdb_url(url):
     return re.match(r'https?://[^/]*imdb\.com/', url)
 
 
+def is_valid_imdb_title_id(value=None):
+    """
+    Return True if `value` is a valid IMDB ID for titles (movies, series, etc).
+    """
+    if not isinstance(value, basestring):
+        return False
```

Comment:
Or you could just do `entry.get('nfo_id', '')` and avoid the None value altogether. It's not the responsibility of this function to hold the developer's hand.